Source/WebGPU/WebGPU/WebGPU.h

```diff
@@ -1334,16 +1334,16 @@ typedef struct WGPURenderPassDescriptor {
     auto colorAttachmentsSpan() const { return unsafeMakeSpan(colorAttachments, colorAttachmentCount); }
 } SWIFT_ESCAPABLE WGPURenderPassDescriptor WGPU_STRUCTURE_ATTRIBUTE;
 
-inline WGPURenderPassTimestampWrites const * _Nullable __counted_by(1) wgpuGetRenderPassDescriptorTimestampWrites(const WGPURenderPassDescriptor * __attribute__((lifetimebound)) __counted_by(1) __attribute__((noescape)) descriptor) {
+inline WGPURenderPassTimestampWrites const * _Nullable __counted_by(1) wgpuGetRenderPassDescriptorTimestampWrites(const WGPURenderPassDescriptor * __counted_by(1) __attribute__((noescape)) descriptor) {
```

Comment:
Seems like `__attribute__((noescape))` is what should be removed here, not `__attribute__((lifetimebound))`.

The function returns `descriptor->timestampWrites`, which means a value from `descriptor` escapes.